frontend/src/Services/Axios.js

```diff
@@ -2,4 +2,8 @@ import Axios from 'axios';
 
 const axios = Axios.create({});
 
+axios.defaults.xsrfHeaderName = "X-CSRFTOKEN";
+axios.defaults.xsrfCookieName = "csrftoken";
```

Comment:
Is this a start to work with `csrftoken`? I also noticed the `csrf_exempt` below.